geoips/interfaces/boundaries.py

```diff
@@ -0,0 +1,9 @@
+from geoips.interfaces.base import BaseInterface, BasePlugin
```

Comment:
These are just place holders until we address #74 , correct?

Reply:
Yes, exactly. Maybe it would be better to just remove this file from this release to avoid confusion?

Reply:
I'm fine leaving them.